lib/bridge/github/contents.rb

```diff
@@ -0,0 +1,20 @@
+class Huboard
+  module Contents
+    ISSUE_TEMPLATE_PATTERN = /^ISSUE_TEMPLATE\.(md|txt)/i
+
+    def issue_template
+      file_list = gh.contents ''
+      return find_template(file_list) if file_list && file_list.size > 0
+
+      file_list = gh.contents '.github'
```

Comment:
As I mentioned below, we should be able to check if `.github` exists in `file_list` before making this API call.